src/proto/filepath.pro

```diff
@@ -56,6 +56,9 @@ int expand_wildcards_eval(char_u **pat, int *num_file, char_u ***file, int flags
 int expand_wildcards(int num_pat, char_u **pat, int *num_files, char_u ***files, int flags);
 int match_suffix(char_u *fname);
 int unix_expandpath(garray_T *gap, char_u *path, int wildoff, int flags, int didstar);
+#if defined(MSWIN) || (defined(UNIX) && !defined(VMS)) || defined(USE_UNIXFILENAME) || defined(PROTO)
```

Comment:
we don't need the ifdefs in the proto files. I think cproto also does not carre about those.

Reply:
Ok I will remove the ifdef.